grumblr/app/assets/javascripts/application.js

```diff
@@ -13,4 +13,20 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
+//= require underscore
+//= require backbone
+//= require handlebars.runtime
+//= require_tree ./templates
+//= require_tree ./routers
+//= require_tree ./models
+//= require_tree ./collections
+//= require_tree ./views
 //= require_tree .
+
+
+$(document).ready(loadGrumbler);
+function loadGrumbler(){
+	mainCollection = new GrumblersCollection();
+	mainCollectionView = new GrumblersListView({collection: mainCollection});
+	mainCollection.fetch();
```

Comment:
Awesome job on this assignment!! :+1: :star:  - re: the trying to remember rails - you did a great job here. Let me know if there are any concpets you'd like to go over.